### ISSUE TITLE
How to use typeorm?

### ISSUE THREAD

**Issue by UrSok** [electron-react-boilerplate/electron-react-boilerplate#2805]
## Summary

I'm trying to use typeorm with sqlite3.
Both are installed fine. sqlite3 is installed as native dependency in src/package.json. typeorm is in package.json.
This is the error I get when I use the following code:

```typescript
  this.connection = await createConnection({
    type: 'sqlite',
    database: path.join('test.sqlite'),
  });
```

![image](https://user-images.githubusercontent.com/5286851/112053457-0bbbab80-8b5d-11eb-92ca-1993bfeaa45f.png)
![image](https://user-images.githubusercontent.com/5286851/112053759-694ff800-8b5d-11eb-83a4-90544ec52d29.png)

**Comment by amilajack** [comment]
I created an example of erb using sqlite: https://github.com/amilajack/erb-sqlite-example. Hope you find this useful

**Comment by am-elkheir** [comment]
> I created an example of erb using sqlite: https://github.com/amilajack/erb-sqlite-example. Hope you find this useful

How did you make the `import sqlite from 'sqlite3';` work. It tells me module not found error which I understand is because `sqlite` is installed in `release/app/node_modules` and not the top level `node_modules`. It works if explicitly imported it from `release/app` but it's annoying.
https://github.com/amilajack/erb-sqlite-example/blob/635e205a59b6e02dfc5ef894f1b4121f5646df5d/src/main/main.ts#L17

**Comment by hua753951hf** [comment]
> I created an example of erb using sqlite: https://github.com/amilajack/erb-sqlite-example. Hope you find this useful

Thank You,Finally solved the problem

**Comment by arimenezes11** [comment]
this throws the "please install sqlite3 package manually" error when i run "npm run start"

**Comment by zifeiyu666** [comment]
@amilajack  I encountered the same problem. Although I can import sqlite directly in main.ts, typeorm cannot detect the existence of sqlite. I need to put typeorm under release/app for it to work.

**Comment by vijaysharma0757** [comment]
I have also faced same issue. Resolved by several steps.
Steps
1. Add sqlite and typeorm in `release/app/package.json` file. Please install only version for sqlite@5.1.6
`npm install sqlite3@5.1.6 --save` 
`npm install typeorm --save`
2.  Remove `sqlite` and `typeorm` from `dependencies` in `package.json` file and add to `devDependencies`
`npm install sqlite3@5.1.6 --save-dev`
`npm install typeorm --save-dev`

Below error resolved by changing sqlite version `5.1.6`
<img width="1255" alt="Screenshot 2024-04-09 at 5 29 48 PM" src="https://github.com/electron-react-boilerplate/electron-react-boilerplate/assets/13532956/065faeda-3083-4722-840c-2e1ba2d405a7">


Below error resolved by step 2. if we add `sqlite` and `typeorm` in `devDependencies`
<img width="980" alt="Screenshot 2024-04-09 at 5 28 44 PM" src="https://github.com/electron-react-boilerplate/electron-react-boilerplate/assets/13532956/43002e0b-5dca-415a-a5ab-82b5751f9b56">
